tests/everest/test_yaml_parser.py

```diff
@@ -17,9 +17,13 @@ def test_random_seed(random_seed):
     if random_seed:
         config["environment"] = {"random_seed": random_seed}
     ever_config = EverestConfig.with_defaults(**config)
-    assert ever_config.environment.random_seed == random_seed
     ert_config = everest_to_ert_config(ever_config)
-    assert ert_config.random_seed == random_seed
+    if random_seed is None:
+        assert ever_config.environment.random_seed > 0
```

Comment:
This seems like a change in behavior? Suggest splitting the changes related to this into a separate commit.